prospector/commit_processor/preprocessor.py

```diff
@@ -49,7 +49,7 @@ def preprocess_commit(git_commit: GitCommit) -> DatamodelCommit:
     result.hunks = git_commit.get_hunks()
     result.hunk_count = len(result.hunks)
     result.message = git_commit.get_msg()
-    result.timestamp = git_commit.get_timestamp()
+    result.timestamp = int(git_commit.get_timestamp())
```

Comment:
I think the best thing it to change the get_timestamp() function in the git.py module